docs/content/reference/migration/migration-0-23.md

```diff
@@ -8,7 +8,16 @@ Previously, you could (confusingly) have two timelines with the same name, as lo
 This is no longer possible.
 Timelines are now uniquely identified by name, and if you use different types on the same timeline, you will get a logged warning, and the _latest_ type will be used to interpret the full set of time data.
 
-## 🐍 Python: replaced `rr.set_time_*` with `rr.set_index`
+## Rename some timeline-related things as "index"
+We're planning on adding support for different types of indices in the future, so to that point we're slowly migrating our API to refer to these things as _indices_ rather than _timelines_.
+
+## Differentiate between timestamps and durations
+We've added a explicit API for setting time, where you need to explicitly specify if a time is either a timestamp (e.g. `2025-03-03T14:34:56.123456789`) or a duration (e.g. `123s`).
+
+Before Rerun would try to guess what you meant (small values were assumed to be durations, and large values were assumes to be durations since the Unix epoch, i.e. timestamps).
```

Comment:
```suggestion
Before, Rerun would try to guess what you meant (small values were assumed to be durations, and large values were assumes to be durations since the Unix epoch, i.e. timestamps).
```